src/components/SignIn/SignIn.js

```diff
@@ -133,25 +116,23 @@ class SignIn extends Component {
                         value={formElement.config.value}
                         invalid={!formElement.config.valid}
                         touched={formElement.config.touched}
-                        changed={(event) => this.inputChangeHandler(event, formElement.id)}                        
+                        changed={(event) => this.inputChangeHandler(event, formElement.id)}
                     />
-                ))} 
-                
-                <button className='Button' disabled={!this.props.formIsValid} onClick={this.submitHandler}>Enter</button>                
-                
+                ))}
+
+                <button className='Button' disabled={!this.state.formIsValid} onClick={this.submitHandler}>Enter</button>
+
             </div>
         )
     }
 }
 
-
 const mapStateToProps = state => (
     {
-        submitted: state.signIn.submitted,
-        cardData: state.cardData,
-        formIsValid: state.signIn.formIsValid
+        signInSlice: state.signInSlice,
```

Comment:
seems you need only `submitted` attribute here

Reply:
done